### DISCUSSION TITLE
"Invalid size returned for Measure" After Upgrading from 0.10 to 11

### DISCUSSION THREAD

**Discussion by furesoft** [AvaloniaUI/Avalonia#15018] [Q&A]
Hi, after a year of break I decided to continue my project. Unfortunatly the upgrade to the newest avalonia had failed and I can't figure it out where the problem is. 

The repo: https://github.com/furesoft/Slithin/tree/develop

The project is kind of complex. Thanks for your help.

**Comment by timunie** [comment]
@furesoft sorry but I believe that this project is too big for the community to debug. Try to isolate the issue in a minimal sample.

**Reply by furesoft** [reply]
I have the same problem in this simpler project: https://github.com/furesoft/FermatCalc
